lib/i18n/_missing_translations.json

```diff
@@ -47,15 +47,15 @@
   "it": {
     "settings": {
       "prefLabels": {
-        "editorToolbarShowInFullscreen": "Show the editor toolbar in fullscreen mode"
+        "editorToolbarShowInFullscreen": "Mostra la barra degli strumenti dell'editor in modalità a schermo intero"
       }
     },
     "appInfo": {
       "dirty": "DIRTY"
```

Comment:
Is this right? /genq

Reply:
Hi,
"DIRTY" is name of version?

Reply:
Dirty builds include changes that are ahead of the last release. It's just a regular adjective